pkg/image/api/types.go

```diff
@@ -194,6 +194,19 @@ type ImageStreamTag struct {
 	unversioned.TypeMeta
 	kapi.ObjectMeta
 
+	// Tag is the spec tag associated with this image stream tag, and it may be null
+	// if only pushes have occured to this image stream.
+	Tag *TagReference
+
+	// Generation is the current generation of the tagged image - if tag is provided
+	// and this value is not equal to the tag generation, a user has requested an
+	// import that has not completed, or Conditions will be filled out indicating any
+	// error.
+	Generation int64
```

Comment:
Spec tags have their own *generation*

On Thu, Feb 11, 2016 at 9:27 AM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/image/api/types.go
> <https://github.com/openshift/origin/pull/6925#discussion_r52607072>:
>
> > @@ -194,6 +194,19 @@ type ImageStreamTag struct {
> >  	unversioned.TypeMeta
> >  	kapi.ObjectMeta
> >
> > +	// Tag is the spec tag associated with this image stream tag, and it may be null
> > +	// if only pushes have occured to this image stream.
> > +	Tag *TagReference
> > +
> > +	// Generation is the current generation of the tagged image - if tag is provided
> > +	// and this value is not equal to the tag generation, a user has requested an
> > +	// import that has not completed, or Conditions will be filled out indicating any
> > +	// error.
> > +	Generation int64
>
> Why can't we use objectmeta's generation?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6925/files#r52607072>.
>